internal/cmd/policy/reorderrules/reorderrules.go

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package reorderrules
+
+import (
+	"strings"
+
+	"github.com/gittuf/gittuf/internal/cmd/common"
+	"github.com/gittuf/gittuf/internal/cmd/policy/persistent"
+	"github.com/gittuf/gittuf/internal/repository"
+	"github.com/spf13/cobra"
+)
+
+type options struct {
+	p          *persistent.Options
+	policyName string
+	ruleNames  string
```

Comment:
Instead of using a string, we could use cobra's StringArray, to take in an array of string, which means if the user mistakenly adds whitespace in front of a rulename, the reorder will still work

Reply:
Thanks, great idea!